scripts/download/cargo-machete.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_dir=$(readlink -f $(dirname $0))
+
+. $script_dir/lib.sh
+
+version=0.5.0
+
+base_url=https://github.com/bnjbvr/cargo-machete/releases/download/v$version
+file_stem=cargo-machete-v$version-$arch_rust-unknown-linux-musl
+
+download_and_decompress \
+    --check-hash sha256 \
+    $base_url/$file_stem.tar.gz \
+    --strip-components 1 $file_stem/cargo-machete
```

Comment:
The [readme](https://github.com/bnjbvr/cargo-machete/) of `cargo-machete` suggests using

```
cargo install cargo-machete
``` 

Do you use this to directly download the binary, or is there another reason? Same question for the other download scripts :)

Reply:
Downloading the pre-compiled binary is much faster than building it from source on every CI run. You may compare how much time it takes to `cargo install cargo-machete` with the time it takes to download. Hint, downloading it takes less then a second:
<img width="982" alt="image" src="https://github.com/rust-marker/marker/assets/36276403/2f05320f-c41f-4757-94f6-e27008b36459">